app/views/goodsProfile/GoodsRecordsView.scala.html

```diff
@@ -56,220 +56,105 @@
 )(implicit request: Request[_], messages: Messages)
 
 
-@preparedTitle = @{
-   if(!appConfig.enhancedSearch) {
-        pagination.items match {
-            case Some(items) if (items.size > 1) => title(form, messages("goodsRecords.title.withPage", page, items.size))
-            case _ => title(form, messages("goodsRecords.title"))
-        }
-    } else {
-        if(searchForm.searchTerm.isEmpty
-            && searchForm.countryOfOrigin.isEmpty
-            && searchForm.statusValue.isEmpty){
-                title(form, messages("goodsRecords.title"))
-        } else {
+
+@layout(
+    pageTitle = {
+        if (!appConfig.enhancedSearch) {
+            pagination.items match {
+                    case Some(items) if items.size > 1 => messages("goodsRecords.title.withPage", page.toString, items.size.toString)
+                    case _ => messages("goodsRecords.title")
+                    }
+                } else {
+              {
                 totalRecords match {
-                    case 0 => title(form, messages("goodsRecords.title.noResults"))
-                    case 1 => title(form, messages("goodsRecords.title.singleRecord"))
-                    case _  =>
-                        pagination.items match {
-                            case Some(items) if (items.size > 1) => title(form, messages("goodsRecords.title.withPageFilter",totalRecords, page, items.size))
-                            case _ => title(form, messages("goodsRecords.title.withoutPageFilter", totalRecords))
```

Comment:
Are we now not missing this case _  withoutPageFilter in the enhanced search bit?